build.gradle

```diff
@@ -31,6 +31,9 @@ sourceSets {
 
 dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.12'
+    compile 'junit:junit:4.12'
+    compile 'junit:junit:4.12'
+    compile 'junit:junit:4.12'
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.